portfolio/src/main/webapp/script.js

```diff
@@ -37,3 +37,143 @@ function deleteReviews() {
     }
   });
 }
+
+/** Creates a map and adds it to the page. */
+function createMap() {
+  const map = new google.maps.Map(
+    document.getElementById('map'), {
+      center: {lat: 42.1784, lng: -87.9979},
+      zoom: 11,
+      styles: [
+        {elementType: 'geometry', stylers: [{color: '#242f3e'}]},
+        {elementType: 'labels.text.stroke', stylers: [{color: '#242f3e'}]},
+        {elementType: 'labels.text.fill', stylers: [{color: '#746855'}]},
+        {
+            featureType: 'administrative.locality',
```

Comment:
Woah are you making your own dark mode? This is cool, but there's not an API setting for it?

Reply:
this is apparently the way to enable dark mode according to the google documentation